### ISSUE TITLE
rest_basic_auth doesn't work properly

### ISSUE THREAD

**Issue by brigadier** [ninenines/cowboy#858]
`cowboy_req:parse_header(<<"authorization">>, Req)` returns `{basic, User, Password}`. In the example match is done against `{<<"basic">>, {User, Password}}`

```
case cowboy_req:parse_header(<<"authorization">>, Req) of
		{<<"basic">>, {User = <<"Alladin">>, <<"open sesame">>}} ->
```

**Comment by essen** [comment]
That's on master or 1.0?

**Comment by brigadier** [comment]
master

**Comment by essen** [comment]
Sounds straightforward enough, can you send a patch?

**Comment by christopheradams** [comment]
This issue looks like it was fixed in 394b4f0, and can therefore be closed.

This exact line is [here](https://github.com/ninenines/cowboy/commit/394b4f0bf7f2a4d496ffe69c8e30cff153d69532#diff-dd07c610236147e51d13b1b439b1c2eaR16).

**Comment by essen** [comment]
Thanks!
